hazelcast/src/main/java/com/hazelcast/internal/config/MemberDomConfigProcessor.java

```diff
@@ -1795,10 +1795,7 @@ private HotRestartConfig createHotRestartConfig(Node node) {
     }
 
     protected void handleCache(Node node) throws Exception {
-        String name = getAttribute(node, "name");
-        CacheSimpleConfig cacheConfig = new CacheSimpleConfig();
-        cacheConfig.setName(name);
-        handleCacheNode(node, cacheConfig);
+        handleCacheNode(node, config.getCacheConfig(getAttribute(node, "name")));
```

Comment:
This is already properly implemented in the YAML parser but wanted to unify the behaviour.